apps/backend/src/eventHandlers/MailService/SMTPMailService.ts

```diff
@@ -174,6 +174,22 @@ export class SMTPMailService extends MailService {
           id: 'isis-rapid-proposal-submitted-uo',
           name: 'ISIS Rapid User Office Submission Email',
         },
+        {
+          id: 'exp-proposal-submitted',
+          name: 'Express PI Co-I Submission Email',
+        },
+        {
+          id: 'exp-proposal-review',
```

Comment:
Is this email the same as the SRA Request email below?

Reply:
No, Receipt of SRA Request is ISIS Sample Safety team.

Reply:
Who is this email to?

Reply:
Principal Investigator (PI) and Co-Investigator (Co-I) details from the previous email sample in production.